app/src/sharedTest/java/org/oppia/android/app/faq/FAQSingleActivityTest.kt

```diff
@@ -104,6 +107,15 @@ class FAQSingleActivityTest {
     Intents.release()
   }
 
+  @Test
+  fun testFAQSingleActivity_hasCorrectActivityLabel() {
```

Comment:
```suggestion
  fun testFaqSingleActivity_hasCorrectActivityLabel() {
```

The activity name itself will need to get updated in a different PR, but we should follow the camel-casing rules defined here: https://developer.android.com/kotlin/style-guide#camel_case.

Reply:
Done. PTAL. Thanks.